clockgating-plugin/tests/Makefile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020-2022 F4PGA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+TESTS = regfile
+include $(shell pwd)/../../Makefile_test.common
```

Comment:
This test does not verify if the plugin does its job correctly. You need to add a regfile_verify target - see other plugins for reference.

Reply:
solved in commit https://github.com/chipsalliance/yosys-f4pga-plugins/pull/382/commits/889b3c28c8c71e632b30441ea862cf809e0f9ba1